docs/features/dead_letter_archive.md

```diff
@@ -19,3 +19,80 @@ In addition to the above, as of Cumulus v9+, the Cumulus API also contains a new
 Sending a POST request to this endpoint will trigger a Cumulus AsyncOperation that will attempt to reprocess (and if successful delete) all Cumulus messages in the dead letter archive, using the same underlying logic as the existing `sfEventSqsToDbRecords`. Otherwise, all Cumulus messages that fail to be reprocessed will be moved to a new archive location under the path `<stackName>/dead-letter-archive/failed-sqs/<YYYY-MM-DD>`.
 
 This endpoint may prove particularly useful when recovering from extended or unexpected database outage, where messages failed to process due to external outage and there is no essential malformation of each Cumulus message.
+
+## Dead Letter Archive Message structure
+
+The Messages yielded to the dead letter archive have some inherent uncertainty to their structure due to their nature as failed messages that may have failed due to structural issues. However there is a standard format that they will overwhelmingly conform to. This follows, but adds attributes to the format documented at [SQSMessage](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_Message.html)
+
+```ts
+{
```

Comment:
it's not in the order that the user would see things, because I don't think the order is guaranteed and this felt like a 'logical' order to me. and I didn't gow ith text formatting because I've put it in a typescript block to get prettier json object with notation
the exact idea here I don't think I can act on, but I'm very open to a better format, i'm just not sure what it is